src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs

```diff
@@ -95,10 +98,20 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
                     queryStringParams.Add("rmode", mode);
                 }
 
+                if (className != null)
+                {
+                    className = "class=\"" + className + "\" ";
+                }
+
+                if (altText != null)
+                {
+                    altText = "alt=\"" + altText + "\" ";
+                }
+
                 content = QueryHelpers.AddQueryString(content, queryStringParams);
             }
 
-            content = "<img src=\"" + content + "\">";
+            content = "<img " + altText + className + "src=\"" + content + "\">";
```

Comment:
Can we use string interpolation here?

Reply:
No, this is fine, and better for performance